app/bundles/DynamicContentBundle/Form/Type/DynamicContentType.php

```diff
@@ -181,17 +291,63 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             );
         }
 
+        $filterModalTransformer = new FieldFilterTransformer($this->translator);
+        $builder->add(
+            $builder->create(
+                'filters',
```

Comment:
This seems to be pulling in a bunch of segment filters that DWC doesn't support such as Goto, email bounced, page hits, etc.

Reply:
The filter list was trimmed down